packages/electron-chrome-web-store/src/browser/installer.ts

```diff
@@ -257,6 +260,14 @@ export async function uninstallExtension(
     session.removeExtension(extensionId)
   }
 
+  if (opts.afterUninstall) {
+    await opts.afterUninstall({
+      id: extensionId,
+      extension: existingExt,
+      manifest: existingExt?.manifest,
+    })
+  }
```

Comment:
Can this be moved into api.ts similar to how `afterInstall` is called? Is there a particular need for this to be called prior to the extension directory being cleaned up?